### PR TITLE
add install requirements.txt packages if exist.

### PR DESCRIPTION
**Pull request by drillan**

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -22,5 +22,10 @@ echo "-----> cloning sphinx document project"
 cd $BUILD_DIR
 git clone $GIT_URL repo
 
+echo "-----> installing 3rd-party python packages"
+if [ -r repo/requirements.txt ]; then
+  pip install -r repo/requirements.txt
+fi
```
Comment (shimizukawa) [inline bin/compile:28]: Can you please update `limitation` https://github.com/shimizukawa/heroku-sphinxbuild-template/blame/master/README.rst#L25

and ChangeLog https://github.com/shimizukawa/heroku-sphinxbuild-template/blob/master/ChangeLog.rst

**Review by shimizukawa — APPROVED** [review]
LGTM! 
can you update documentation?

File: bin/compile (hunk 1 above)
Comment (drillan) [reply bin/compile:28]: done

File: ChangeLog.rst [hunk 2]
```diff
@@ -12,3 +12,7 @@ First release
 * Serving HTML files that is generated with sphinx-build.
 * BASIC authentication for readers if you want.
 
+Release 2019.07.31
+==================
+
+* Install 3rd-party python packages if requirements.txt is exist at source directory.
```
Comment (shimizukawa) [inline ChangeLog.rst:18]: Thanks!

Newer should placed at first.
I'll move it after merging.

File: README.rst [hunk 3]
```diff
@@ -17,20 +17,18 @@ Features
 * Building your sphinx document on git repository some where.
 * Serving HTML files that is generated by sphinx-build.
 * Serving HTML files with BASIC authentication for readers if you want.
+* Install 3rd-party python packages / sphinx extensions (You need requirements.txt at top of the source directory.)
```
Comment (shimizukawa) [inline README.rst:20]: 👍

**Review by shimizukawa — APPROVED** [review]
LGTM! Thanks for your contribution!

**Pull request merged**